notebooks/L1_working_with_age_ensembles.ipynb

```diff
@@ -25,9 +25,10 @@
     "\n",
```

Comment:
<p>"pass in a vector for paleo data values" --&gt; or a matrix, no? Isn't the idea that there can be matrices for chron AND/OR paleo values?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LinkedEarth/PyleoTutorials/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Addressed in [b83a608](https://github.com/LinkedEarth/PyleoTutorials/pull/39/commits/b83a60874afed075ce7b2dcc49b71af93035f8ad)